### PR TITLE
Bridge Pangolin Parachain Message and Code Cleaning

### PR DESCRIPTION
**Pull request by furoxr**
Base on https://github.com/darwinia-network/darwinia-common/pull/1117

Add bridge messages pallet into pangolin

- Add pallet_bridge_messages instance for pangolin-parachain
- Add pallet_bridge_dispatch instance for pagnolin-parachain
- Update bridged_balance_to_this_balance parameters

TODO:
- [x] [Fee market](https://github.com/darwinia-network/darwinia-common/blob/c6c88b1759d345614301882156e5568d83d853cc/node/runtime/pangolin/src/pallets/bridge_pangolin_parachain_messages.rs#L52-L53)
- [ ] [Dispatch call filter](https://github.com/darwinia-network/darwinia-common/blob/c6c88b1759d345614301882156e5568d83d853cc/node/runtime/pangolin/src/pallets/bridge_dispatch.rs#L48-L49)

### CODE REVIEW

File: node/runtime/pangolin/src/bridges/substrate/pangolin_parachain_messages.rs [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+	RuntimeDebug,
+};
+use pallet_bridge_messages::EXPECTED_DEFAULT_MESSAGE_LENGTH;
+use sp_runtime::{traits::Zero, FixedPointNumber, FixedU128};
+use sp_std::ops::RangeInclusive;
+// --- darwinia-network ---
+use crate::{bridges::pangoro_messages::Pangolin, *};
+
+/// Identifier of PangolinParachain in the relay chain.
+pub const PANGOLIN_PARACHAIN_ID: u32 = 2071;
```
Comment (HackFisher) [inline node/runtime/pangolin/src/bridges/substrate/pangolin_parachain_messages.rs:34]: Pangolin parachain's para id is `2105`?

**Review by HackFisher — COMMENTED** [review]
We can add the fee market together in this PR.

**Comment by HackFisher** [comment]
@AurevoirXavier Please note that we also need this for Pangolin.

**Comment by AurevoirXavier** [comment]
Prefer to finish the #1166 first.
Replacing the remaining `*_primitives` to `bp_*`.

**Review by AurevoirXavier — APPROVED** [review]

**Comment by AurevoirXavier** [comment]
Please review the runtime configurations carefully.

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**